lib/index.js

```diff
@@ -1,4 +1,14 @@
+import Avatar from '../src/components/Avatar/Avatar';
```

Comment:
Can we import these from lib so there's one less thing to remember to add when creating a new component?

Reply:
I'm starting to think these should be automatically generated by the build script. I've seen other component libraries do it.

Reply:
I think automatically generated would be good, we can look at that later though...

I will swap to lib now...